### ISSUE TITLE
Updating to 3.62.2 causes errors with `.default` properties

### ISSUE THREAD

**Issue by aredridel** [braintree/braintree-web#504]
### General information

* SDK version: 3.62.2
* Environment: Sandbox
* Browser and OS: Any

### Issue description

Installing 3.62.2, compiled with rollup.js into our build, I get errors relating to package default imports:

```
Uncaught (in promise) TypeError: i.default is not a function
    emit framebus.js:45
    emit index.js:71
    _cleanUpFocusIntercepts hosted-fields.js:730
    removeExtraFocusElements focus-change.js:80
    removeExtraFocusElements focus-change.js:74
    _cleanUpFocusIntercepts hosted-fields.js:729
    HostedFields hosted-fields.js:623
    promise callback*HostedFields hosted-fields.js:617
    1 index.js:260
    promise callback*create$1 index.js:259
    wrapPromise wrap-promise.js:18
    createHostedFields payment-step.js:299  // Our code below here in this backtrace
    step_loaded payment-step.js:39
    CheckoutController checkout-controller.js:49
    _rollbar_wrapped rollbar.min.js:1
    dispatch checkout-step.js:39
    api_call checkout-step.js:179
    action_finish shipping-step.js:128
    CheckoutStep checkout-step.js:25
    _rollbar_wrapped rollbar.min.js:1
    o rollbar.min.js:2
    CheckoutStep checkout-step.js:20
    ShippingStep shipping-step.js:16
    execute shipping-step.js:152
    i system-core.js:274
    e system-core.js:270
    C system-core.js:219
```

Pardon the systemjs chaff here, but I think it provides context.

`i.default` in this case is the code referring to  the `framebus` package, which is updated to version 4. Version 4 publishes some ESM-derived code that has a `.default` property on what's returned from require — in some bundlers. With rollup, this is not the case and the code fails. Switching to esm-based modules is unfortunately a breaking change, it seems, with the state of the ecosystem.

**Comment by crookedneighbor** [comment]
:/ thanks for the report. We'll see what we can do in framebus to work around this.

**Comment by aredridel** [comment]
Good luck. This stuff is super gross right now — it might actually mean bundling it into braintree-web to fix it.

**Comment by crookedneighbor** [comment]
@aredridel Can you provide a sample rollup config? I'm having trouble reproducing the error you are getting. This is mine:

```js
import commonjs from '@rollup/plugin-commonjs';
import resolve from '@rollup/plugin-node-resolve';

export default {
  input: 'main.js',
  output: {
    format: 'umd',
    file: 'public/rollup.js'
  },
  plugins: [
    commonjs(),
    resolve()
  ]
};
```

Not sure if you have particular plugins you're using, or a particular format, or what.

**Comment by crookedneighbor** [comment]
Never mind, I was just able to reproduce it.

**Comment by crookedneighbor** [comment]
OK, I think we can solve this by simply using the `export =` typescript syntax instead of `export default`. Can you provide me a rollup config to try? I want to make sure it solves your problem before publishing a fix.

**Comment by aredridel** [comment]
```
import alias from "@rollup/plugin-alias";
import commonjs from "@rollup/plugin-commonjs";
import json from "@rollup/plugin-json";
import replace from "@rollup/plugin-replace";
import resolve from "@rollup/plugin-node-resolve";
import { terser } from "rollup-plugin-terser";
import svelte from "rollup-plugin-svelte";

const aliases = {
  entries: [{ find: "cuid", replacement: require.resolve("cuid/dist/cuid.js") }]
};

require("dotenv").config();

const env = process.env.APP_ENV ? process.env.APP_ENV : "production";

const config = {
  output: {
    dir: "public/js/",
    sourcemap: true,
    entryFileNames: "[name]-[hash].[format].js",
    chunkFileNames: "[name]-[hash].[format].js",
    preferConst: true,
    format: "system"
  },
  plugins: [
    alias(aliases),
    svelte({
      dev: env != "production",
      hydratable: true
    }),
    resolve({
      preferBuiltins: false
    }),
    commonjs(),
    replace({
      "process.env.NODE_ENV": JSON.stringify(env),
      "process.env.NODE_DEBUG": JSON.stringify(null)
    }),
    json(),
    env == "production" ? terser() : null,
  ]
};

export default config;
```

**Comment by crookedneighbor** [comment]
I tested my fix with your config, and it works!

The only changes I made to the config were to change the input and output, remove the aliases bit, and changed the format from `system` to `umd`. Of those changes, the only one that I imagine may make a difference is the `format` config. I'm not too familiar with rollup, and can't find any solid documentation on what `format: 'system'` is supposed to do. All I know is that it added a `System` global that the browser couldn't find when I tried to run the script.

Hoping to get a fix out for this sometime this week.
